apps/common/utils/index.tsx

```diff
@@ -99,3 +99,10 @@ export const graphFetcher = async (args: [string, string]): Promise<GraphQLRespo
 export const formatPercent = (n: number, min = 2, max = 2): string => `${formatAmount(n || 0, min, max)}%`;
 
 export const formatUSD = (n: number, min = 2, max = 2): string => `$ ${formatAmount(n || 0, min, max)}`;
+
+export const toNormalizedBN = (value?: BigNumberish, decimals?: number): TNormalizedBN => ({
```

Comment:
It's not ideal to have a function take two optional params like this, we should refactor for named params instead
```suggestion
export const toNormalizedBN = ({value, decimals}: {value?: BigNumberish, decimals?: number}): TNormalizedBN => ({
```